release_files/install.sh

```diff
@@ -23,19 +23,28 @@ if command -v sudo > /dev/null && [ "$(id -u)" -ne 0 ]; then
     SUDO="sudo"
 fi
 
-get_latest_release() {
+if [ -z ${NETBIRD_RELEASE+x} ]; then 
```

Comment:
This seems to be failing. Can you try the following?

```suggestion
if [ "x-$NETBIRD_RELEASE" == "x-" ]; then 
```